src/Servers/IIS/AspNetCoreModuleV2/InProcessRequestHandler/inprocessapplication.cpp

```diff
@@ -165,8 +165,8 @@ IN_PROCESS_APPLICATION::LoadManagedApplication(ErrorContext& errorContext)
         // If server wasn't initialized in time shut application down without waiting for CLR thread to exit
         errorContext.statusCode = 500;
         errorContext.subStatusCode = 37;
-        errorContext.generalErrorType = "ANCM Failed to Start Within Startup Time Limit";
-        errorContext.errorReason = format("ANCM failed to start after %d milliseconds", m_pConfig->QueryStartupTimeLimitInMS());
+        errorContext.generalErrorType = "ASP.NET Core Module Failed to Start Within Startup Time Limit";
```

Comment:
```suggestion
        errorContext.generalErrorType = "ASP.NET Core app failed to start within startup time limit";
```

It's the app that's failing to start, right? Even if it is a part of the framework causing it, I think it's clearer to say "app" here.